tutorials/space/activity4.md

```diff
@@ -149,27 +216,33 @@ game.onUpdateInterval(5000, function () {
         5 . . . . . . . . . . . . . . 5
         5 5 5 5 5 5 5 5 5 5 5 5 5 5 5 5
         `, 0, 50)
-    fuel.setKind(SpriteKind.Gas)
-    fuel.x = randint(0, 160)
+    myFuel.x = randint(0, 160) 
+    // @highlight  
+    myFuel.setKind(SpriteKind.Gas)
 })
 ```
 
+
 ## Step 7
-Drag a ``||sprites:on overlaps||`` block into the workspace. Change the second
-sprite kind to ``||sprites:Gas||``.
+When your ship overlaps fuel, you'll want the gas to disappear as the tank refills.
+<hr/>  
 
-```blocks
-namespace SpriteKind {
-    export const Gas = SpriteKind.create()
-}
+🔲 Drag an ``||sprites:on [sprite] of kind [Player] overlaps [othersprite] of kind [Player]||`` 
+container into the workspace. 
 
-sprites.onOverlap(SpriteKind.Player, SpriteKind.Gas, function (sprite, otherSprite) {
-})
-```
+🔲 Change the last argument from ``||sprites:Player||`` to ``||sprites:Gas||``.  
+
+🔲 To refill the status bar after grabbing fuel, snag a ``||statusbars:set [statusbar] [value] to [0]||`` block 
+and snap it in to your newest **overlaps** container.  Change the value from **0** to **100**.
+
+🔲 Finally, make sure the used fuel disappears by snapping a ``||sprites:destroy [mySprite] ⊕||`` block 
+into the bottom of the same **overlaps** container and replacing
+``||variables:mySprite||`` with ``||variables:otherSprite||``
+
+![Grabbing variable from block](/img/space/give-var.gif "So that's how you do that!")
```

Comment:
```suggestion
![Grabbing variable from block](https://raw.githubusercontent.com/microsoft/pxt-skillmap-sample/main/img/space/give-var.gif "So that's how you do that!")
```